crypto/fipsmodule/bn/asm/x86-mont.pl

```diff
@@ -288,8 +282,8 @@
 	&jle	(&label("outer"));
 
 	&emms	();				# done with mmx bank
-
-}	# The non-SSE2 code was removed. 
+	&jmp	(&label("common_tail"));
```

Comment:
I made this change just so the DIFF between *ring* and BoringSSL would be minimal.